pydatastructs/miscellaneous_data_structures/tests/test_queue.py

```diff
@@ -1,9 +1,18 @@
-from pydatastructs.miscellaneous_data_structures import Queue
+from pydatastructs.miscellaneous_data_structures import (
+    Queue, ArrayQueue, LinkedListQueue)
 from pydatastructs.utils.raises_util import raises
 
 def test_Queue():
+    q1 = Queue(implementation='array', items=[0])
+    q1 = Queue()
+    q1 = Queue(implementation='linked_list')
 
+def test_ArrayQueue():
+    q = ArrayQueue()
+    q1 = Queue()
+    raises(IndexError, lambda: q1.popleft())
     q1 = Queue(implementation='array', items=[0])
+    assert type(q) == type(q1)
```

Comment:
Use `_check_type`.

Reply:
```suggestion
    assert _check_type(q1, ArrayQueue)
```

Reply:
both `_check_type(q1, Queue)` and `_check_type(q1, ArrayQueue)` will give True. The main thinking behind the assertion was to check if the object is of type Queue or of type ArrayQueue.

Reply:
So basically it is to be verified that correct implementation queue is created. Well then for now we can go for using `type` for now for this case.